python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1188,3 +1188,10 @@ def get_log_filenames():
         else None
         for i, each_mr in _per_device_mrs.items()
     }
+
+
+def get_free_device_memory(percent):
```

Comment:
Since the name of the function is not `get_percent_of_free_device_memory`, do you think that the parameter should default to 100?

Reply:
FWIW, I think `get_percent_of_free_device_memory` would be a more misleading name, since that to me indicates that the result is some $n \in [0, 100]$. But the return value is actually some number of bytes $b \in [0, \text{free\\_device\\_memory} \times \text{percent}]$.

Reply:
I would call that `get_percent_of_device_memory_that_is_free`. :)